meson.build

```diff
@@ -0,0 +1,34 @@
+project('lgi', 'c',
+  version: '0.9.2',
+  meson_version: '>= 0.44.0',
```

Comment:
`meson` should warn now when you use too new of features (maybe not for everything as that's relatively new). I can't tell you off-hand since I wrote this months ago rather quickly. It may have been an arbitrary version.